### DISCUSSION TITLE
BUG: Audio player shows up on Safari, Safari Mobile, and Chrome Mobile, but not Chrome web

### DISCUSSION THREAD

**Discussion by bryanwhiting** [quarto-dev/quarto-cli#8615] [Q&A] [answered]
### Description

TLDR, the `![](audio.m4a)` shows up with a nice audio player in safari and on mobile, but not on Chrome desktop for some reason.

([Blog post](https://www.bryanwhiting.com/tech/embedding-audio-in-quarto.html) where I discuss this, and [blog post](https://www.bryanwhiting.com/creative/guitar-clouds) where I do the embedding)

On Safari Desktop:
![image](https://github.com/quarto-dev/quarto-cli/assets/18710907/1e479069-40fd-4db3-aab6-6a413f16f5f6)

On Chrome Desktop:
![image](https://github.com/quarto-dev/quarto-cli/assets/18710907/2723fb43-28d8-4957-b508-013e3eadc979)

**Comment by cscheid** [comment]
I think we're just creating a link element (`<a href="..."></a>`) to that file, in which case there's nothing Quarto is doing especially, and I'm not sure what you were expecting us to do.

**Reply by mcanouil** [reply, marked as answer]
FYI, that's all Pandoc, you don't even need the file (the file name simply needs to be recognise as an audio file).

<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/7b37606c-5423-40fb-b69d-f9ba1473a3ef">

Pandoc will convert to the following HTML, then it's up to the browser to read it.

```html
<p>This is a playground for Quarto.</p>
<p><audio src="audio.m4a" controls=""><a href="audio.m4a">Audio</a></audio></p>
```

Note that I am using Brave (chromium based) and it's working, be sure to check your local configuration as I highly doubt Chrome is faulty here.

<img width="1285" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/a416f682-2905-4fa5-9c22-2207a6acc59b">

**Reply by bryanwhiting** [reply]
There should be two types of audio players, no?

When I wrap in the quarto shortcode it creates a video player (the big black one you see).

In this screenshot below the top audio player is created using the markdown syntax, with the annotation appearing on the right margin. 

The bottom player is using a video shortcode.

![image](https://github.com/quarto-dev/quarto-cli/assets/18710907/3dc8d5cb-364b-47c6-96a2-e4f7048fbe47)

It's just that the top audio player (the one that looks better) doesn't appear in Chrome. If I look at your bottom screenshot, it also doesn't appear, as there should be two. Am I seeing that right?

**Comment by bryanwhiting** [comment]
Oh, I think Quarto is doing what it should.

When I link with `![](audio.mp4a)` quarto turns it into an `<audio>` tag. This brings up an audio player everywhere except on my Desktop Chrome.

what about my local config might I need to check?

- Bryan
________________________________

**Reply by mcanouil** [reply]
I don't know but that's out of Quarto support scope I am afraid.
